scripts/superdesk/notification/notification.js

```diff
@@ -96,23 +97,26 @@
      */
     NotifyConnectionService.$inject = ['$rootScope', 'notify', 'gettext', '$timeout', 'session'];
     function NotifyConnectionService($rootScope, notify, gettext, $timeout, session) {
-        var successTimeout, alertTimeout;
         var _this = this;
         _this.message = null;
 
         $rootScope.$on('disconnected', function(event) {
-            _this.message = 'Disconnected to Notification Server, attempting to reconnect ...';
-            $timeout.cancel(alertTimeout);
-            alertTimeout = $timeout(function() {
-                notify.error(gettext(_this.message));
-            }, 100);
+            if (!notify.disconnectionNotified) {
+                _this.message = 'Disconnected to Notification Server!';
```

Comment:
gettext call should be here, otherwise it won't be picked when extracting strings